excercise-e-shop/home.html

```diff
@@ -0,0 +1,356 @@
+          <i class="icon"></i>
+        </button>
+        <button class="cart">
+          <i class="icon"></i>
+        </button>
+        <button class="user">
+          <i class="icon"></i>
+        </button>
+      </div>
+    </div>
+  </header>
+  <main>
+    <section class="banner">
+      <p class="banner-title">Sale of the <span>summer</span>collection</p>
+      <a href="#" class="shopping">
+        <i class="icon"></i>
+        <p class="shopping-text">Shop now</p>
+      </div>
+      <ul class="grid-list">
+        <li class="grid-item footer-banner">
+          <div class="footer-banner-item">
+            <i class="icon"></i>
+            <div class="footer-banner-detail">
+              <h4 class="footer-banner-name">Free Shipping</h4>
+              <p class="footer-banner-description">On purchases over $199</p>
+            </div>
+          </div>
+        </li>
+        <li class="grid-item footer-banner">
+          <div class="footer-banner-item">
+            <i class="icon"></i>
+            <div class="footer-banner-detail">
+              <h4 class="footer-banner-name">99% Satisfied Customers</h4>
+              <p class="footer-banner-description">Our clients' opinions speak for themselves</p>
+            </div>
+          </div>
+        </li>
+        <li class="grid-item footer-banner">
+          <div class="footer-banner-item">
+            <i class="icon"></i>
+            <div class="footer-banner-detail">
+              <h4 class="footer-banner-name">Originality Guaranteed</h4>
+              <p class="footer-banner-description">30 days warranty for each product from our store</p>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section>
+      <div class="container">
+        <ul class="grid-list categories">
+          <li class="grid-item category">
+            <div class="category-item">
+              <img src="#" alt="category">
+              <h4 class="category-name">New arrivalsare now in!</h4>
+              <a href="#" class="btn btn-primary">SHOW COLLECTION</a>
+            </div>
+          </li>
+          <li class="grid-item category">
+            <div class="category-item">
+              <img src="#" alt="category">
+              <h4 class="category-name">New arrivalsare now in!</h4>
+              <a href="#" class="btn btn-primary">MORE DETAILS</a>
+            </div>
+          </li>
+          <li class="grid-item category">
+            <div class="category-item">
+              <img src="#" alt="category">
+              <h4 class="category-name badge-sale">Sale this summer</h4>
+              <a href="#" class="btn btn-primary">VIEW ALL</a>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <div class="container">
+        <div class="recommend-cloth-title">
+          <h4 class="text-title">Selected just for you</h4>
+          <a href="#" class="btn btn-secondary">SHOW MORE</a>
```

Comment:
`SHOW MORE` should be `position: absolute` from this `section` due to this action show more information for all section, not section-header